internal/crossdock/server/oneway/server.go

```diff
@@ -74,7 +75,8 @@ func Start() {
 	h := &onewayHandler{http.NewTransport()}
 	dispatcher.Register(raw.OnewayProcedure("echo/raw", h.EchoRaw))
 	dispatcher.Register(json.OnewayProcedure("echo/json", h.EchoJSON))
-	dispatcher.Register(onewayserver.New(h))
+	dispatcher.Register(onewayserver.New(&thriftHandler{h}))
+	dispatcher.Register(crossdockpb.BuildOnewayYarpcProcedures(&protoHandler{h}))
```

Comment:
Question: is the `BuildOnewayYarpcProcedures` name generated? If not, might be better as `BuildOnewayYARPCProcedures`, in keeping with Go's [policy on initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).